src/fetch/payouts.py

```diff
@@ -137,6 +137,15 @@ def reward_scaling(self) -> Fraction:
         The reward is multiplied by this factor"""
         return 1 - SERVICE_FEE_FACTOR * self.service_fee
 
+    def total_service_fee(self) -> Fraction:
+        """Scaling factor for service fee
+        The reward is multiplied by this factor"""
+        return (
+            SERVICE_FEE_FACTOR
+            * self.service_fee
+            * (self.primary_reward_cow + self.quote_reward_cow)
```

Comment:
Can this be negative? If so, should we maybe redefine now how we measure the service fee collected?

Reply:
It can be negative. I created a draft PR #424 for changing the behavior.